ntbs-service/Pages/Index.cshtml

```diff
@@ -86,12 +86,18 @@
                                 <nhs-table-item heading-text="Dismiss">
                                     @if (!item.NotDismissable)
                                     {
-                                        <form method="post" action="@RouteHelper.GetAlertPath(@item.AlertId, AlertSubPaths.Dismiss)">
-                                            @Html.AntiForgeryToken()
-                                            <button class="button-link nhsuk-link govuk-link--no-visited-state govuk-link" aria-label="Dismiss alert">
-                                                X
-                                            </button>
-                                        </form>
+                                        <single-submit-form inline-template>
+                                            <form method="post"
+                                                  action="@RouteHelper.GetAlertPath(@item.AlertId, AlertSubPaths.Dismiss)"
+                                                  v-on:submit="disableButton">
+                                                @Html.AntiForgeryToken()
+                                                <button class="button-link nhsuk-link govuk-link--no-visited-state govuk-link"
+                                                        aria-label="Dismiss alert"
+                                                        ref="submitButton">
+                                                    X
+                                                </button>
+                                            </form>
+                                        </single-submit-form>
```

Comment:
It would be good if this were a partial, as it's nearly identical to the view on the notification overview page. However, I'd like to get this ticket into the release so I'm going to leave that as tech debt (and create a ticket in the backlog).